sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/Uri.java

```diff
@@ -230,12 +249,31 @@ public int getPriority() {
     }
 
     static class HealthStatusAndDiagnosticStringTuple {
-        private final String diagnsoticString;
         private final HealthStatus status;
+        private final String ATTEMPTING = "Attempting";
+        private final String NOT_ATTEMPTING = "NotAttempting";
```

Comment:
Should be static final (and in that case probably needs ot be moved to the parent class). Otherwise it wastes 64bit per Uri instance.

Reply:
Moved them to parent class and made them static.